packages/gatsby-plugin-graphql-config/src/gatsby-node.ts

```diff
@@ -12,7 +12,7 @@ async function cacheGraphQLConfig(program: IStateProgram): Promise<void> {
         schema: resolve(base, `.cache/schema.graphql`),
         documents: [
           resolve(base, `src/**/**.{ts,js,tsx,jsx,esm}`),
-          resolve(base, `.cache/fragments.graphql`),
+          `./.cache/fragments.graphql`,
```

Comment:
I'm confused why this works. Isn't .cache in th program directory, even in a monorepo?

Reply:
yes, so this is resolving relative to the graphql-config file noted in the README

Reply:
actually, let's hold off on this. there's a lot of reasons this won't work. thanks for the feedback!